workshop/content/exercises/trigger-pipeline.adoc

```diff
@@ -29,7 +29,7 @@ spec:
   type: image
   params:
   - name: url
-    value: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/spring-petclinic
+    value: %image_registry%/pipelines-tutorial/spring-petclinic
```

Comment:
Can we include the namespace dynamically here as well (i.e., replace `pipelines-tutorial` with the namespace created for the user)?